### DISCUSSION TITLE
Editing images and layout

### DISCUSSION THREAD

**Discussion by PeeWee-Sweden** [arkypita/LaserGRBL#2247] [General]
I'm tearing my hair here. I've added several images to a layout, and now I want to move them around on the layout. But I can't find any way to do that. Am I supposed to re-import every single image, and manage to place them in the layout by changing their offset manually? Are you kidding me?? Please tell me I'm just missing something!

**Comment by Tribo-2dot0** [comment]
Você pode montar o layout em Inkscape e exportar a página como png
